en/references/judges/README.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Judges
+description: "Overview of all judges available on Dodona"
+---
+
+# Judges
+
+This guide is written for teachers who are creating exercises for Dodona.
+
+Before you create an exercise, you should decide which `judge` you want to use.
+The `judge` is the program that will evaluate the submissions of students.
+It is often written for one specific programming language, or for a set of programming languages.
+Each `judge` has its own configuration options, we will provide a link to the relevant documentation for each `judge` below.
+These judge specific options should be provided in the `evaluation` directory of the [exercise configuration](/en/references/exercise-directory-structure).
+
+Advanced users can also create their own judge, see [this guide](/en/guides/creating-a-judge).
+
+Dodona currently supports the following judges:
+
+## TESTed
+TESTED is a whitebox judge that can be used for multiple programming languages.
+It uses a simple custom test format, that is indpendent of the programming language.\
+**Programing languages:** Bash, C, C#, Haskell, Java, Javascript, Kotlin, Python\
```

Comment:
Do we want to do this like this, or should we look into a definition list, e.g. https://github.com/markdown-it/markdown-it-deflist?

Reply:
As discussed I will ignore this one, because it does not really improve the visuals, and makes each judge take up a lot more space
![image](https://user-images.githubusercontent.com/21177904/218117641-2c09d22d-fb79-4f65-9fcc-46226a64b3e9.png)